README.md

```diff
@@ -1 +1,62 @@
-# space-travel-hub
\ No newline at end of file
+# space-travel-hub
+
+This project is geared towards working with APIS to build a space travel hub through the following practices:
+
+- Create repository withe readme file
+- Create Branch
+- Applying best practices for HTML, CSS and JavaScript
+- Applying Github Flow
+- Applying Git Flow
+- Set Up Webpack
+- Descriptive and Understanding Commit Messages and ReadMe file
+- Pull Request in Github for code review
+- Use kanban board to plan and allocate tasks for the developers
+- Using the URLs provided missions and rockets
+- Add fetch Data functionality from APIs
+- Add Stylings
+- Carry jest test
+- Deployment
+
+## For Access Online
+
+Click: https://efoosag.github.io/space-travel-hub/
+
+## Built With
+
+- Major languages
+  HTML
+  CSS
+  JavaScript
+  React and Redux
+
+- Technologies used
+  GitHub and Git
+  Visual Studio Code
+
+### Prerequisites
+
+    Laptop or Deskstop Computer
+    Internet Connection
+    Power Supply
+
+## Authors
+
```

Comment:
- [ ] Hello excellent work on the React capstone project but please add **Getting started section** and add all the directions on how to copy the repo and add the appropriate npm packages and how to start the app and run the tests also.